src/category_theory/Fintype.lean

```diff
@@ -100,4 +111,18 @@ noncomputable def is_skeleton : is_skeleton_of Fintype skeleton skeleton.incl :=
 { skel := skeleton.is_skeletal,
   eqv := by apply_instance }
 
+/-- Finite types are given the discrete topology. -/
+protected def discrete_topology (A : Fintype) : topological_space A := ⊥
+
 end Fintype
+
+section discrete_topology
+
+local attribute [instance] Fintype.discrete_topology
+
+/-- The natural functor from `Fintype` to `Profinite`, endowing a finite type the discrete topology. -/
+def Fintype_to_Profinite : Fintype ⥤ Profinite :=
+{ obj := λ A, ⟨⟨A⟩⟩,
+  map := λ _ _ f, ⟨f⟩ }
+
+end discrete_topology
```

Comment:
Might it not make more sense to put this in `topology.category.Profinite`?

Reply:
moved!